source/installation/howto_docker.md

```diff
@@ -27,19 +27,19 @@ Let's say that you want to install the [openfisca-country-template](https://gith
         cd my-openfisca # Updated with the real path to your working directory
         ```
 
-2. Build a container with Python 3.7, Git and console commands.
+2. Build a container with Python 3.9, Git and console commands.
```

Comment:
Shouldn't this be 3.11 now? 🤔

Reply:
Yes, I am however going to remove the changes to this file from this PR as it is covered extensively in the [update-default-install](https://github.com/openfisca/openfisca-doc/tree/update-default-install) branch now. This specific issue is addressed: https://github.com/openfisca/openfisca-doc/pull/306/files#diff-0e2a0ce6def61a81e4ee35c5ab8876b3e6f57ce8ea1553e589b551a482e62291R31